server/src/main/java/module-info.java

```diff
@@ -419,7 +420,8 @@
     provides org.apache.lucene.codecs.PostingsFormat
         with
             org.elasticsearch.index.codec.bloomfilter.ES85BloomFilterPostingsFormat,
-            org.elasticsearch.index.codec.bloomfilter.ES87BloomFilterPostingsFormat;
+            org.elasticsearch.index.codec.bloomfilter.ES87BloomFilterPostingsFormat,
+            ES812PostingsFormat;
```

Comment:
Trivially, would you mind fully qualifying the name here, and removing the import. It will help improve readability of the provides.

Reply:
That was my naughty IDE. Added back the full path.